lib/src/main/java/com/springboot/webpush/common/api/NotificationOptions.java

```diff
@@ -3,9 +3,11 @@
 import java.util.List;
 
 /**
- * The action class, specific properties to be used to send the {@link PushMessage} to the service.
+ * The {@link NotificationOptions} class, specific properties to be used to send the {@link PushMessage} to the service.
+ * Currently not used, and marked as deprecated - possible future usage.
  */
-public class Action {
+@Deprecated
+public class NotificationOptions {
```

Comment:
Yeah, I think we can just get rid of this

Reply:
Done.